packages/pigment-css-react-new/README.md

```diff
@@ -0,0 +1,29 @@
+# Pigment CSS
```

Comment:
Content like this would be useful on the docs.

Reply:
Do mean something in addition to the [styling](https://deploy-preview-390--pigment-css.netlify.app/features/styling) guide ?